.travis.yml

```diff
@@ -1,10 +1,83 @@
-language: node_js
-node_js: stable
+_aliases:
+  - &node
+    addons:
+      firefox: latest-beta
+    language: node_js
+    node_js: stable
+    before_install: skip
+    before_script: skip
 
-addons:
-  firefox: latest-beta
+jobs:
+  include:
+    - stage:
+      language: python
+      python: 3.6
+      addons:
+        firefox: latest-nightly
```

Comment:
Is Nightly the earliest version that support Gecko driver and such? Ideally we'd be testing on 57.0.

Reply:
Do we just want to test on 57?